src/resources/ui/nautilus-search-popover.ui

```diff
@@ -145,7 +145,7 @@
                     <property name="transition_duration">250</property>
                     <child>
                       <object class="GtkScrolledWindow">
-                        <property name="height_request">200</property>
+                        <property name="height_request">150</property>
```

Comment:
Can we drop this since we are dropping composite mode so no longer need to support extremely low resolutions?

Reply:
Package being moved to OBS.
Dropped in the OBS request: https://obs-master.endlessm-sf.com/request/show/19953